pyaedt/modules/PostProcessor.py

```diff
@@ -876,7 +917,8 @@ def available_display_types(self, report_category=None):
             report_category = self.available_report_types[0]
         if report_category:
             return list(self.oreportsetup.GetAvailableDisplayTypes(report_category))
-        return []
+        else:  # pragma: no cover
+            return []
```

Comment:
You should not need to add the `else` statement.
```suggestion
            return []  # pragma: no cover
```